docs/resources/opa_constraint.md

```diff
@@ -21,13 +21,14 @@ resource "rafay_opa_constraint" "tfdemoopaconstraint1" {
     project = "terraform"
   }
   spec {
+    template_name = "request-limit-ratio-template"
     artifact {
+      type = "Yaml"
       artifact {
         paths {
           name = "file://artifacts/one/request-limit-ratio-constraint.yaml"
```

Comment:
Can you replace "one" with "request-limit-ratio" (name of the constarint) int the file path?

Reply:
done